telos-frontend/src/components/widget-drawer/WidgetDrawer.jsx

```diff
@@ -0,0 +1,29 @@
+import './WidgetDrawer.css';
+import { useState } from 'react';
+
+const Arrow = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" height="24" viewBox="0 0 24 24" width="24">
```

Comment:
Is there a material-ui icon that could be used instead of this?

Reply:
There is. It's called Chevron Left. https://material-ui.com/components/material-icons/ I'll go make a low-priority issue.